tests/conftest.py

```diff
@@ -590,6 +590,14 @@ def pytest_addoption(parser) -> None:
         help="check non closed threads i started during test",
     )
 
+    # Run profiler tests only
+    parser.addoption(
+        "--profiling",
+        action="store_true",
+        default=False,
+        help="run profiling tests only.",
+    )
```

Comment:
If run with the `--profiling` flag, only profiler tests will run. If the flag is not present, all the other tests will be run.